apps/store/src/services/PriceForm/PriceForm.helpers.ts

```diff
@@ -0,0 +1,94 @@
+import { SE_HOUSE } from './data/SE_HOUSE'
+import { InputField } from './Field.types'
+import { Form, FormSection, JSONData, Template } from './PriceForm.types'
+
+const TEMPLATES: Record<string, Template | undefined> = {
+  SE_HOUSE: SE_HOUSE,
+}
+
+export const fetchPriceTemplate = (id: string) => {
+  return TEMPLATES[id]
+}
+
+export const convertTemplate = (template: Template): Form => {
```

Comment:
nit: 

```suggestion
export const convertTemplateIntoForm = (template: Template): Form => {
```

[Reference](https://github.com/ryanmcdermott/clean-code-javascript#function-names-should-say-what-they-do)